### PR TITLE
Add state management quickstart

### PR DESCRIPTION
**Pull request by nirmash**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

**Comment by paulyuk** [comment]
Wow, thank you for this contribution @nirmash !!  I'm adding @greenie-msft, @amulyavarote  and @hhunter-ms to review.

**Review by paulyuk — DISMISSED** [review]
One quick thing I see is the tests/checks are failing on docker.com/*.  I see that oftentimes with rate limiting, because so many tests run anonymously on github and they get rate limited.

There are a couple solutions.
- mitigate temporarily by removing the links
- work with @amulyavarote to make sure this check is first doing a docker login so we're no longer anonymous

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+### Step 1: Set up the environment
+
+Clone the sample we've provided.
+
+```bash
+git clone https://github.com/dapr/quickstarts.git
```
Comment (paulyuk) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md:32]: One small ask.  Can we add a note under here or on line 33 with a link pointing to the github repo itself?  I think it's inconvenient that all of our quickstarts only let you see code if you first clone it.

**Review by paulyuk — COMMENTED** [review]
Just a few comments but LGTM

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+linkTitle: "State Management"
+weight: 70
+description: "Get started with Dapr's State Store"
+---
+
+Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}). In this quickstart, you will learn how to store and query data as key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}).
+
+State management is one of the most common needs of any application: new or legacy, monolith or microservice. Dealing with different databases libraries, testing them, handling retries and faults can be time consuming and hard.
+
+Dapr provides state management capabilities that include consistency and concurrency options. In this guide we’ll start of with the basics: Using the key/value state API to allow an application to save, get and delete state.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md:13]:
```suggestion
Dapr provides state management capabilities that include consistency and concurrency options. In this quickstart, you'll use the basic key/value state API to allow an application to save, get, and delete state. 
```

**Comment by amulyavarote** [comment]
Hi! Thanks for the contribution.

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,624 @@
+---
+type: docs
+title: "Quickstart: State Management"
+linkTitle: "State Management"
+weight: 70
+description: "Get started with Dapr's State Store"
+---
+
+Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}). In this quickstart, you will learn how to store and query data as key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}).
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}).  You can use Dapr's state management consistency and concurrency capabilities to:
* Maintain and test different databases and libraries.
* Handle retries and faults.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+
+State management is one of the most common needs of any application: new or legacy, monolith or microservice. Dealing with different databases libraries, testing them, handling retries and faults can be time consuming and hard.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion

```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md (hunk 2 above)
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
In this quickstart, you use the key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}) and allow an application to save, get, and delete state. Our example below uses a Redis instance, but you can use RabbitMQ, Kafka, etc.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,624 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Python 3.7+ installed](https://www.python.org/downloads/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+### Step 1: Set up the environment
+
+Clone the sample we've provided.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Clone the [sample we've provided in our quickstarts repo](https://github.com/dapr/quickstarts/tree/master/state_management).
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 6]
```diff
@@ -0,0 +1,624 @@
...
+pip3 install -r requirements.txt
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- python3 app.py
+```
+
+In the `order-processor` service, we're writing, reading and deleting an orderId key \ value pair to the Redis instance called `statestore` [(as defined in the `statestore.yaml` component)]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 7]
```diff
@@ -0,0 +1,624 @@
...
+    logging.info('Result after get: ' + str(result.data))
+
+    # Delete state from the state store
+    client.delete_state(store_name=DAPR_STORE_NAME, key=orderId)
+    logging.info('Deleting Order: %s', order)
+```
+
+### Step 3: View the order-processor outputs
+
+Notice, as specified in the code above, the code saves application state in the Dapr stat store, reads it and then deletes it.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```

**Review by amulyavarote — APPROVED** [review]

**Review by amulyavarote — CHANGES_REQUESTED** [review]
Please fix the validation errors and the PR comments

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 8]
```diff
@@ -0,0 +1,624 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Latest Node.js installed](https://nodejs.org/download/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 1: Set up the environment
+
+Clone the sample we've set up:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Clone the [sample we've provided in our quickstarts repo](https://github.com/dapr/quickstarts/tree/master/state_management).
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 9]
```diff
@@ -0,0 +1,624 @@
...
+
+- `package.json`
+- `package-lock.json`
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- npm run start
+```
+In the `order-processor` service, we're writing, reading and deleting an orderId key \ value pair to the Redis instance called `statestore` [(as defined in the `statestore.yaml` component)]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 10]
```diff
@@ -0,0 +1,624 @@
...
+// Delete state from the state store
+client.state.delete(STATE_STORE_NAME, orderId.toString());    
+result.then(function(val) {
+    console.log("Deleting Order: ", val);
+});
+
+```
+### Step 3: View the order-processor outputs
+
+Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md:200]:
```suggestion
Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 11]
```diff
@@ -0,0 +1,624 @@
...
+var result = await client.GetStateAsync<string>(DAPR_STORE_NAME, orderId.ToString());
+Console.WriteLine("Getting Order: " + result);
+
+// Delete state from the state store
+await client.DeleteStateAsync(DAPR_STORE_NAME, orderId.ToString());
+Console.WriteLine("Deleting Order: " + order);
+```
+### Step 3: View the order-processor outputs
+
+Notice, as specified in the code above, the code saves application state in the Dapr stat store, reads it and then deletes it.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 12]
```diff
@@ -0,0 +1,624 @@
...
+
+        // Delete state from the state store
+        client.deleteState(DAPR_STATE_STORE, String.valueOf(orderId)).block();
+        LOGGER.info("Deleting Order: " + orderId);
+        TimeUnit.MILLISECONDS.sleep(1000);
+    }
+```
+### Step 3: View the order-processor outputs
+
+Notice, as specified in the code above, the code saves application state in the Dapr stat store, reads it and then deletes it.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 13]
```diff
@@ -0,0 +1,624 @@
...
+  fmt.Println("Getting Order: " + string(result.Value))
+
+  // Delete state from the state store
+  _ = client.DeleteState(ctx, STATE_STORE_NAME, strconv.Itoa(orderId))
+  log.Print("Deleting Order: " + string(order))
+```
+
+### Step 3: View the order-processor outputs
+
+Notice, as specified in the code above, the code saves application state in the Dapr stat store, reads it and then deletes it.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Notice, as specified in the code above, the code saves application state in the Dapr state store, reads it, then deletes it.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 14]
```diff
@@ -0,0 +1,624 @@
...
+dotnet build
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- dotnet run
+```
+
+In the `order-processor` service, we're writing, reading and deleting an orderId key \ value pair to the Redis instance called `statestore` [(as defined in the `statestore.yaml` component)]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 15]
```diff
@@ -0,0 +1,624 @@
...
+mvn clean install
+```
+
+Run the `order-processor` publisher service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components -- java -jar target/order-processor-0.0.1-SNAPSHOT.jar
+```
+
+In the `order-processor` service, we're writing, reading and deleting an orderId key \ value pair to the Redis instance called `statestore` [(as defined in the `statestore.yaml` component)]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 16]
```diff
@@ -0,0 +1,624 @@
...
+go build app.go
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components -- go run app.go
+```
+
+In the `order-processor` service, we're writing, reading and deleting an orderId key \ value pair to the Redis instance called `statestore` [(as defined in the `statestore.yaml` component)]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 17]
```diff
@@ -0,0 +1,624 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [.NET SDK or .NET 6 SDK installed](https://dotnet.microsoft.com/download).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 1: Set up the environment
+
+Clone the sample we've set up:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Clone the [sample we've provided in our quickstarts repo](https://github.com/dapr/quickstarts/tree/master/state_management).
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 18]
```diff
@@ -0,0 +1,624 @@
...
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- Java JDK 11 (or greater):
+  - [Oracle JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html#JDK11), or
+  - [OpenJDK](https://jdk.java.net/13/)
+- [Apache Maven](https://maven.apache.org/install.html), version 3.x.
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 1: Set up the environment
+
+Clone the sample we've provided.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Clone the [sample we've provided in our quickstarts repo](https://github.com/dapr/quickstarts/tree/master/state_management).
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 19]
```diff
@@ -0,0 +1,624 @@
...
+
+For this example, you will need:
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started).
+- [Latest version of Go](https://go.dev/dl/).
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+
+### Step 1: Set up the environment
+
+Clone the sample we've provided.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
Clone the [sample we've provided in our quickstarts repo](https://github.com/dapr/quickstarts/tree/master/state_management).
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
As mentioned earlier, add changes from the Python section to the rest of the languages for consistency.

**Review by amulyavarote — APPROVED** [review]

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 20]
```diff
@@ -0,0 +1,624 @@
...
+pip3 install -r requirements.txt
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- python3 app.py
+```
+
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 21]
```diff
@@ -0,0 +1,624 @@
...
+
+- `package.json`
+- `package-lock.json`
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- npm run start
+```
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 22]
```diff
@@ -0,0 +1,624 @@
...
+dotnet build
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- dotnet run
+```
+
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 23]
```diff
@@ -0,0 +1,624 @@
...
+mvn clean install
+```
+
+Run the `order-processor` publisher service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components -- java -jar target/order-processor-0.0.1-SNAPSHOT.jar
+```
+
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 24]
```diff
@@ -0,0 +1,624 @@
...
+go build app.go
+```
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components -- go run app.go
+```
+
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
The `order-processor` service writes, reads, and deletes an `orderId` key/value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop.
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 25]
```diff
@@ -0,0 +1,624 @@
...
+
+```go
+  // Save state into the state store
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md:537]:
```suggestion
client, err := dapr.NewClient()

  // Save state into the state store
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 26]
```diff
@@ -0,0 +1,624 @@
...
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- dotnet run
+```
+
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
+
+```cs
+// Save state into the state store
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md:303]:
```suggestion
var client = new DaprClientBuilder().Build();

// Save state into the state store
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 27]
```diff
@@ -0,0 +1,624 @@
...
+
+Run the `order-processor` service alongside a Dapr sidecar.
+
+```bash
+dapr run --app-id order-processor --components-path ../../../components/ -- npm run start
+```
+The `order-processor` service writes, reads, and deletes an `orderId` key\value pair to the `statestore` instance [defined in the `statestore.yaml` component]({{< ref "#statestoreyaml-component-file" >}}). As soon as the service starts, it performs a loop:
+
+```js
+// Save state into the state store
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]:
```suggestion
const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT);

// Save state into the state store
```

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 28]
```diff
@@ -0,0 +1,624 @@
...
+description: "Get started with Dapr's State Store"
+---
+
+Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}). You can use Dapr's state management consistency and concurrency capabilities to:
+* Maintain and test different databases and libraries.
+* Handle retries and faults.
+
+In this quickstart, you use the key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}) and allow an application to save, get, and delete state. Our example below uses a Redis instance, but you can use RabbitMQ, Kafka, etc.
+
+<img src="/images/state-management-overview.png" width=800 style="padding-bottom:15px;">
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]: Use updated diagram from @greenie-msft

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
just a few more changes - thank you!

File: daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md [hunk 29]
```diff
@@ -0,0 +1,624 @@
...
+linkTitle: "State Management"
+weight: 70
+description: "Get started with Dapr's State Store"
+---
+
+Let's take a look at Dapr's [State Management building block]({{< ref state-management >}}). You can use Dapr's state management consistency and concurrency capabilities to:
+* Maintain and test different databases and libraries.
+* Handle retries and faults.
+
+In this quickstart, you use the key/value pairs in the [supported state stores]({{< ref supported-state-stores.md >}}) and allow an application to save, get, and delete state. Our example below uses a Redis instance, but you can use RabbitMQ, Kafka, etc.
```
Comment (hhunter-ms) [inline daprdocs/content/en/getting-started/quickstarts/statemanagement-quickstart.md]: Also - could you please capitalize "Quickstarts" throughout the doc? thanks!

File: daprdocs/content/en/getting-started/quickstarts/_index.md [hunk 30]
```diff
@@ -24,7 +24,7 @@ Hit the ground running with our Dapr quickstarts, complete with code samples aim
 | ----------- | ----------- |
 | [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) | Invoke a method using HTTP proxying with Dapr's Service Invocation building block. |
 | [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Get started with Dapr's Publish and Subscribe building block. |
-| State Management   | Coming soon. |
+| [State Management]({{< ref statemanagement-quickstart.md >}}) | Create a microservice to demonstrate Dapr's state management API. |
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/_index.md]:
```suggestion
| [State Management]({{< ref statemanagement-quickstart.md >}}) | Create stateful applications using the state management API. |
```

File: daprdocs/content/en/getting-started/quickstarts/_index.md [hunk 31]
```diff
@@ -24,7 +24,7 @@ Hit the ground running with our Dapr quickstarts, complete with code samples aim
 | ----------- | ----------- |
 | [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) | Invoke a method using HTTP proxying with Dapr's Service Invocation building block. |
 | [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Get started with Dapr's Publish and Subscribe building block. |
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/_index.md]:
```suggestion
| [Publish and Subscribe]({{< ref pubsub-quickstart.md >}}) | Send and receive messages using the publish and subscribe API. |
```

File: daprdocs/content/en/getting-started/quickstarts/_index.md [hunk 32]
```diff
@@ -23,8 +23,8 @@ Hit the ground running with our Dapr quickstarts, complete with code samples aim
 | Quickstarts | Description |
 | ----------- | ----------- |
 | [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) | Invoke a method using HTTP proxying with Dapr's Service Invocation building block. |
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/quickstarts/_index.md]:
```suggestion
| [Service Invocation]({{< ref serviceinvocation-quickstart.md >}}) |  Call a method on another service using the service invocation API. |
```

**Review by msfussell — APPROVED** [review]

**Pull request merged**